cmd/notation/verify.go

```diff
@@ -35,6 +35,19 @@ func verifyCommand(opts *verifyOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "verify [reference]",
 		Short: "Verifies OCI Artifacts",
+		Long: `
+** This command is in preview and under development. **
+Prerequiste: a trusted certificate needs to be generated using command "notation cert". 
+
+Example - Verify a signature using the trusted certificate added to the trusted store:
```

Comment:
Trusted store is not in verify command workflow yet. Suggest to remove this example

Reply:
Removed `added to the trusted store` to avoid ambiguity.